spec/model/like_spec.rb

```diff
@@ -17,7 +17,7 @@
     it 'belongs to an author (User)' do
       association = described_class.reflect_on_association(:author)
       expect(association.macro).to eq(:belongs_to)
-      expect(association.foreign_key).to eq(:author_id)
+      expect(association.foreign_key).to eq(author_id)
       expect(association.class_name).to eq('User')
     end
```

Comment:
- [ ] Great job working on the failing tests, but the issue is still available. I would advise you to empty the database when running the tests, perhaps you are testing using data in your database.

![test](https://github.com/serengia/rails-blog-app/assets/53124548/c198fa76-ec23-4924-9db3-4688771ea59b)

Reply:
Well resolved ✔️ 

![image](https://github.com/serengia/rails-blog-app/assets/38088818/e7ccd37a-17a2-43bb-8d35-be06e1f36f2d)